### PR TITLE
Update em-cave pages

### PR DESCRIPTION
**Pull request by bpdanskin**
Updates to correct the cave-related errors in the data book. Simplifies some of the code outputs by making them code blocks. To run this successfully, the build environment requires em-specific packages: 'caveclient', 'standard_transform' and 'skeleton_plot'

### CODE REVIEW

File: databook/anatomy/microns-em/em-querying-tables.md [hunk 1]
```diff
@@ -127,7 +130,7 @@ We can get the whole table as a DataFrame:
 ```{code-cell}
 :tags: [remove-stderr]
 
-cell_type_df = client.materialize.tables.aibs_soma_nuc_metamodel_preds_v117().query()
+cell_type_df = client.materialize.tables.aibs_metamodel_celltypes_v661().query()
```
Comment (JEElsner) [inline databook/anatomy/microns-em/em-querying-tables.md:133]: Right above this in the markdown we say we're accessing `aibs_soma_nuc_metamodel_preds_v117`, can we update this text to be consistent with the code?

**Review by JEElsner — CHANGES_REQUESTED** [review]
Code looks good! Just one comment on the narrative explaining the code.

File: databook/anatomy/microns-em/em-querying-tables.md (hunk 1 above)
Comment (bpdanskin) [reply databook/anatomy/microns-em/em-querying-tables.md:133]: Yes! Good catch.

**Review by bpdanskin — COMMENTED** [review]
updated table name in the text

**Review by JEElsner — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**